README.md

```diff
@@ -1,2 +1,3 @@
 # training
 Documentation and workshop materials for IIIF training
+[View the gitbook](https://iiif.github.io/training/intro-to-iiif/).
```

Comment:
Do we want to list other docs under this Training repo as well? There is the [Mirador "Getting Started" guide](http://projectmirador.org/docs/docs/getting-started.html), but maybe that's not officially IIIF?